person.rb

```diff
@@ -1,21 +1,24 @@
+require './rentals'
+
 class Nameable
   def correct_name
     raise NotImplementedError
   end
 end
 
 class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
```

Comment:
Accordance to: a person has many rentals,
- [ ] Setters for `@rentals` should be present.

Reply:
@medaminedev66 Okay.. added